lib/grpc.ex

```diff
@@ -1,8 +1,10 @@
 defmodule GRPC do
+  @ver GRPC.Mixfile.project[:version]
```

Comment:
I was not sure if it would overlap with function or if there is reserved attribute with that name.